src/handlers.ts

```diff
@@ -0,0 +1,28 @@
+import { Context, ScheduledEvent } from 'aws-lambda';
+import * as daylightSavingsChangeReminder from './daylightSavingsChangeReminder';
+import * as leaveReminder from './leaveReminder';
+import { Notifier } from './notifier';
+import * as publicHolidayReminder from './publicHolidayReminder';
+
+export const weeklyReport = async (
```

Comment:
🐘 you can actually type these as `ScheduledHandler` to avoid having to define the unused params & return (i.e `export const weeklyReport: ScheduledHandler = async ()  => { ... }`)

Reply:
That's neat, will do.